caas/kubernetes/provider/cloud.go

```diff
@@ -141,7 +140,10 @@ func UpdateKubeCloudWithStorage(k8sCloud *cloud.Cloud, storageParams KubeCloudSt
 				// And no preferred storage classes with expected annotations found.
 				//  - workloadStorageClassAnnotationKey
 				//  - operatorStorageClassAnnotationKey
-				return "", UnknownClusterError{CloudName: cloudType}
+				return "", UnknownClusterError{
+					Message:   "suitable Kubernetes storage class for workload storage not found in cluster",
```

Comment:
Could we also include an extra line telling people an option would be to kubectl create a default storage class, ie how to fix the issue.
The error message could be defined as a multi-line string var like the add-k8s ones and just assigned to Message